Upstream: yes
Reason: Fixes a cmake warning

From 3a7b5207b458f2ba4e146a23e1eef00d15e576ff Mon Sep 17 00:00:00 2001
From: Lukas W <lukaswhl@gmail.com>
Date: Wed, 11 Jul 2018 21:56:50 +0200
Subject: [PATCH] CMake: Fix mismatching if condition error

Regression from c45a21aac3beb0bba4c1796cf591ea44d2f2bc46
---
 cmake/modules/FindSndio.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/FindSndio.cmake b/cmake/modules/FindSndio.cmake
index 3fcd71360..e993702c2 100644
--- a/cmake/modules/FindSndio.cmake
+++ b/cmake/modules/FindSndio.cmake
@@ -29,6 +29,6 @@ find_package_handle_standard_args(SNDIO DEFAULT_MSG SNDIO_LIBRARY SNDIO_INCLUDE_
 if(SNDIO_FOUND)
     set(SNDIO_INCLUDE_DIRS "${SNDIO_INCLUDE_DIR}")
     set(SNDIO_LIBRARIES "${SNDIO_LIBRARY}")
-endif(HAVE_SNDIO)
+endif()
 
 mark_as_advanced(SNDIO_INCLUDE_DIR SNDIO_LIBRARY SNDIO_INCLUDE_DIRS SNDIO_LIBRARIES)
-- 
2.20.1

